editor/editor_properties.h

```diff
@@ -487,14 +487,17 @@ class EditorPropertyQuaternion : public EditorProperty {
 	AcceptDialog *warning_dialog = nullptr;
 
 	Label *euler_label = nullptr;
+	VBoxContainer *normalize_quaternion = nullptr;
 	VBoxContainer *edit_custom_bc = nullptr;
 	EditorSpinSlider *euler[3];
 	Button *edit_button = nullptr;
+	Button *normalize_quaternion_bttn = nullptr;
```

Comment:
```suggestion
	Button *normalize_quaternion_btn = nullptr;
```
Nit pick for consistency with other classes.